Configs/CircleCI.xcconfig

```diff
@@ -0,0 +1,3 @@
+#include "Base.xcconfig"
```

Comment:
I've included `Base.xcconfig` here because it exists but to be honest I'm not sure if we actually use this. Does Xcode use it by default if it's called `Base.xcconfig`?

Reply:
Looks like it is...it's easily spotted in build settings.

<img width="1552" alt="screen shot 2019-02-28 at 1 50 48 pm" src="https://user-images.githubusercontent.com/387596/53601035-fcc1c500-3b5f-11e9-9110-d1c9fa06ecf1.png">

Reply:
Ah thanks